### PR TITLE
added functionality for download as csv

### PR DESCRIPTION
**Pull request by codiebeulaine**
@miltontony ready for review

### CODE REVIEW

**Comment by miltontony** [comment]
looks, good.Test please

**Comment by codiebeulaine** [comment]
@miltontony test written, review please

File: molo/yourwords/admin.py [hunk 1]
```diff
@@ -15,6 +17,20 @@
...
+def download_as_csv(YourWordsCompetitionEntryAdmin, request, queryset):
+    opts = queryset.model._meta
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment;filename=export.csv'
+    writer = csv.writer(response)
+    field_names = [field.name for field in opts.fields]
+    writer.writerow(field_names)
+    for obj in queryset:
+        writer.writerow([getattr(obj, field) for field in field_names])
+    print response
```
Comment (miltontony) [inline molo/yourwords/admin.py]: stray print, please remove

**Comment by miltontony** [comment]
:+1: :cake:

**Pull request merged**